parsers/IN.py

```diff
@@ -279,7 +279,8 @@ def fetch_state_consumption(session, state):
 
         for future in concurrent.futures.as_completed(futures):
             total_consumption += future.result()
-
+    if total_consumption == 0:
+        raise ParserException(parser="IN.py", message=f"{target_datetime}: No valid consumption data found", zone_key=zone_key)
```

Comment:
Instead of doing it in the parser it might make sense to do it in the TotalConsumptionList class so it's used for other zones as well. Parsed consumption should never be 0.

Reply:
Good point, let's do that